website/content/sdk/guides/intros/stocks/fundamental-analysis.md

```diff
@@ -1,131 +1,243 @@
 ---
 title: Fundamental Analysis
-keywords:
-  [
-    "stocks", "stock", "options", "option", "call", "put", "earnings", "calendar", "how-to", "guide", "scripts", "fundamental", "analysis", "technical", "behavioural", "analyst", "equity", "research", "api", "sdk", "application", "python", "notebook", "jupyter",
-  ]
-excerpt: "This guide introduces the FA menu in the context of the OpenBB SDK."
 ---
 
-The FA module provides programmatic access to the commands from within the OpenBB Terminal. Import the OpenBB SDK module, and then access the functions similarly to how the Terminal menus are navigated. The code completion will be activated upon entering `.`, after, `openbb.fa`
-​
+The FA module provides programmatic access to the commands from within the OpenBB Terminal. To get the most out of these functions, we recommend obtaining free API keys from:
```

Comment:
Why do we say "free" here? But say some are premium only - lets just say "we recommend obtainign API keys from:"  - we should also link to the page to get the API keys

Reply:
Done